build.cake

```diff
@@ -1236,20 +1235,37 @@ T GetBuildVariable<T>(string key, T defaultValue)
     return Argument(key, EnvironmentVariable(key, upperCaseReturnValue));
 }
 
-void StartVisualStudio(string sln = "./Microsoft.Maui.sln")
+void StartVisualStudio(string sln = "./Microsoft.Maui.sln", bool usePreviewVs = false)
 {
     if(isCIBuild)
         return;
 
     if(IsRunningOnWindows())
     {
-        StartProcess("powershell",
-            new ProcessSettings
-            {
-                Arguments = new ProcessArgumentBuilder()
-                    .Append("start")
-                    .Append(sln)
-            });
+        if(usePreviewVs)
+        {
+            var vsLatest = VSWhereLatest(new VSWhereLatestSettings { IncludePrerelease = true, });
```

Comment:
Eilon was doing this on the powershell side here: https://github.com/dotnet/maui/pull/806

Should we pick a side? powershell or cake?